connect/runtime/src/test/java/org/apache/kafka/connect/util/KafkaBasedLogTest.java

```diff
@@ -313,6 +314,13 @@ public void run() {
                 // Already have FutureCallback that should be invoked/awaited, so no need for follow up finishedLatch
             }
         });
+
+        //this consumer.poll is to invoke the task scheduled by "consumer.schedulePollTask"
+        try {
+            consumer.poll(0);
```

Comment:
I left a comment about why the poll should not be removed in the KafkaBaseLog class, but having to add it here should be a warning sign something is wrong to -- that consumer is internal to the class, so there is no way for anything external to invoke `consumer.poll()`. If the test needs to do that, the class cannot function properly when used in the actual application.